packages/tradershub/src/features/cfd/screens/ChangePassword/InvestorPassword/MT5ChangeInvestorPasswordInputsScreen.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { Fragment } from 'react';
+import { Formik } from 'formik';
+import { useTradingPlatformInvestorPasswordChange } from '@deriv/api';
+import { Button, Text, TextField } from '@deriv/quill-design';
+import { ActionScreen } from '../../../../../components';
+import { Provider } from '@deriv/library';
+import useDevice from '../../../hooks/useDevice';
+import { PlatformDetails } from '../../../constants';
+
+type TFormInitialValues = {
+    currentPassword: string;
+    newPassword: string;
+};
+
+type TProps = {
+    sendEmail?: VoidFunction;
+    setNextScreen?: VoidFunction;
+};
+
+const MT5ChangeInvestorPasswordInputsScreen: React.FC<TProps> = ({ sendEmail, setNextScreen }) => {
```

Comment:
what about React.FC? I remember @aizad-deriv shared article we don't need to use it in tradershub. Or maybe something changed again?
cc: @aizad-deriv